### ISSUE TITLE
Configurable `baseName`

### ISSUE THREAD

**Issue by dan-lee** [k-tomoyasu/react-native-oss-license#47]
Shouldn't the `baseName` be configurable instead of hardcoded?

https://github.com/k-tomoyasu/react-native-oss-license/blob/871cc43c1cd4e630fc9a4ad7db54c24e71823b13/src/formatter/ios/SettingsBundle.ts#L4

**Comment by k-tomoyasu** [comment]
@dan-lee 
Thank you for your feedback!
It looks like to make a command option. 
Here is an idea.
````
--plist-prefix (only format=setting-bundle) prefix of plist filename.
```

If you have any other ideas for naming the option, I'd be happy to hear.

**Comment by dan-lee** [comment]
What about `bundleId`? I think that's a common name for Android/iOS apps.

**Comment by k-tomoyasu** [comment]
Ok, How about following option?
```
--bundle-id unique id of your app.  It is used for output such as "plist" filename, etc. (default: null)
```

**Comment by dan-lee** [comment]
I like it! 👏

**Comment by k-tomoyasu** [comment]
👍 
This feature will be included in the next version!(release near future)

**Comment by k-tomoyasu** [comment]
It have published as version `0.5.0`! Thanks 👍

**Comment by dan-lee** [comment]
Yay, thank you a lot @k-tomoyasu :)!
